server/coordinator/procedure/storage.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 type Write interface {
-	CreateOrUpdate(ctx context.Context, meta Meta) error
+	CreateOrUpdate(ctx context.Context, meta Meta, ttlSec int64) error
```

Comment:
How about adding a new method called `CreateOrUpdateWithTTL` and keep the `CreateOrUpdate` same as before? If so, there is no need to change the irrelevant codes and there is no need to use zero ttlSec as a sentinel for no ttl.